### PR TITLE
Update daylite to 6.5.5

### PR DESCRIPTION
**Pull request by core-code**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/daylite.rb [hunk 1]
```diff
@@ -1,14 +1,20 @@
 cask 'daylite' do
-  version '6.6'
-  sha256 'dde1ebdc27b4a63c763c901004520401c984a4c1fb06c2a40a93ef5b11095b8b'
+  if MacOS.version <= :mojave
```
Comment (reitermarkus) [inline Casks/daylite.rb]: `<= :high_sierra`

**Review by reitermarkus — CHANGES_REQUESTED** [review]

File: Casks/daylite.rb (hunk 1 above)
Comment (core-code) [reply Casks/daylite.rb]: thanks ;)

**Comment by core-code** [comment]
not sure about travis failure

File: Casks/daylite.rb (hunk 1 above)
Comment (core-code) [reply Casks/daylite.rb]: @reitermarkus i think i've addressed the changes you've requested. also, i don't understand the travis failure as those two are in uninstall launchctl. any ideas? thanks

**Comment by core-code** [comment]
can someone have a look at this?

**Comment by core-code** [comment]
@reitermarkus i've updated to 6.6.2. can you have a look?

**Comment by reitermarkus** [comment]
This needs a rebase.

**Pull request closed without merging**